vim/vim
# Fix: 'cursorline' with visualmode redraw issue

How to reproduce:
- Start Vim with some options
  $ vim --clean +"set cursorline" +"call setline(1, repeat(['100'], 100))"
- Start Visual mode linewise and page down and cursor ups and down ...
  V\<C-F>kkkjk

Expected behavior:
- Visual mode Highlighting is fine.

Actrual behavior:
- The wrong highlight is displayed below the line with the cursor.

Solution:
- Even if it does not match a certain condition, update `wp->w_last_cursorline`.

--
Best regards,
Hirohito Higashi (h_east)
